scripts/dice-wfrp4e.js

```diff
@@ -54,11 +54,19 @@ class DiceWFRP
       slBonus: dialogOptions.data.slBonus || 0,
       successBonus: dialogOptions.data.successBonus || 0,
     });
-    mergeObject(cardOptions,
-    {
-      user: game.user._id,
-      sound: CONFIG.sounds.dice
-    })
+
+    if(cardOptions.title.includes('Alcohol') || dialogOptions.rollOverride && dialogOptions.rollOverride.name == "weaponOverride"){
+      mergeObject(cardOptions,
+        {
+          user: game.user._id,
+        })
+    } else {
+      mergeObject(cardOptions,
+        {
+          user: game.user._id,
+          sound: CONFIG.sounds.dice
+        })
+    }
```

Comment:
Could this be clearer? I'm not sure what's going on here.
Also needs localization.